rpc/src/v1/impls/eth.rs

```diff
@@ -545,6 +546,55 @@ impl<C, SN: ?Sized, S: ?Sized, M, EM, T: StateInfo + 'static> Eth for EthClient<
 		Box::new(future::done(res))
 	}
 
+	fn proof(&self, _address: RpcH160, values:Vec<RpcH256>, num: Trailing<BlockNumber>) -> BoxFuture<EthAccount> {
```

Comment:
nit: missing a space in `values:` and `Vec`.

Reply:
fixed